src/components/molecules/NavSearchBar/NavSearchBar.scss

```
@@ -1,8 +1,19 @@
 @import "scss/constants.scss";
 
+$search-results-width: 18.125rem;
+
 .nav-search-links {
```

Match the filename/component it relates to:

```suggestion
.NavSearchBar {
```